src/main.ts

```diff
@@ -1,9 +1,23 @@
 import { NestFactory } from '@nestjs/core';
+import { DocumentBuilder, SwaggerModule } from '@nestjs/swagger';
 import { AppModule } from './app.module';
 
-async function bootstrap() {
+async function bootstrap(): Promise<void> {
+  const prefix = 'api/v1';
+  const port = process.env.NLIMS_MEDIATOR_PORT || 3000;
   const app = await NestFactory.create(AppModule);
-  app.setGlobalPrefix('nlims/api/v0/fhir');
```

Comment:
When creating a channel in the IL, the base path that will separate this mediator is /nlims/api/v0/*.* or something like that. Case in point, it just cannot be /api/v1/ as that would bring about conflicts with the other channels.

Reply:
Okay.